Foundation/FileManager+Win32.swift

```diff
@@ -264,8 +264,9 @@ extension FileManager {
     }
 
     internal func _createSymbolicLink(atPath path: String, withDestinationPath destPath: String) throws {
-        let faAttributes: WIN32_FILE_ATTRIBUTE_DATA = try windowsFileAttributes(atPath: path)
-        var dwFlags: DWORD = DWORD(SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE)
+        var dwFlags = DWORD(SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE)
+        // destPath may not exist in which case we default to creating a file symlink
+        let faAttributes = (try? windowsFileAttributes(atPath: destPath)) ?? WIN32_FILE_ATTRIBUTE_DATA()
```

Comment:
We need to resolve the final path here ... `destPath` needs to be resolved relative to `path`.

Reply:
Let's continue to throw here; we don't know what the target of the symlink will be and we can silently fail later with a symlink being a file symlink with a directory target.